config/config_test.go

```diff
@@ -899,3 +899,13 @@ func TestThrottlerDefaultConfig(t *testing.T) {
 	assert.NoError(t, err)
 	defer closeCloser(t, closer)
 }
+
+func TestWithRandomNumber(t *testing.T) {
+	cfg := &Configuration{
+		ServiceName: "test-random-number",
+	}
+	randomNum := func() uint64 { return 1 }
+	_, closer, err := cfg.NewTracer(WithRandomNumber(randomNum))
+	assert.NoError(t, err)
+	defer closeCloser(t, closer)
```

Comment:
I think that this test should also assert that a created span has the expected trace/span ids.

Currently it only checks that creating a new tracer doesn't return error.